content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,139 @@
+kind: Documentation
+further_reading:
+    - link: 'tracing/profiling/trace_profile'
+      tag: 'Documentation'
+      text: 'Connect traces and profiles.'
+    - link: 'tracing/profiling/code_level_metrics'
+      tag: 'Documentation'
+      text: 'View code level metrics.'  
+    - link: 'tracing/profiling/getting_started'
+      tag: 'Documentation'
+      text: 'Enable continuous profiling for your application.'
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+{{< img src="tracing/profiling/search_profiles.gif" alt="Search profiles by tags">}}
+
+Each row is a profile of a process for a short amount of time. By default profiles are uploaded once a minute. Depending on the language, these processes are profiled between 15s and 60s.
+
+You can filter according to infrastructure tags or application tags set up from your [environment tracing configuration][1]. By default the following facets are available:
+
+| Facet    | Definition                                                                                             |
+| -------- | ------------------------------------------------------------------------------------------------------ |
+| Env      | The environment your application is running on (`prod`, `staging`).                                    |
+| Service  | The name of the [service][2] your code is running.                                                     |
+| Version  | The version of your code.                                                                              |
+| Host     | The host name your profiled process is running on. |
+| Runtime  | The type of runtime the profiled process is running (`JVM`, `CPython`).                                |
+| Language | The language of your code (`Java`, `Python`).                                                          |
+
+The following measures are available:
+
+| Measure           | Definition                                                                                                                                                                           |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| CPU cores              | CPU load summed on all cores of the process. |
+| Memory Allocation | Memory allocation rate in `Bytes/s` over the course of the profile. This value can be above the maximum amount of RAM memory because it can be garbage collected during the profile. |
+
+
+
+## Profiles
+
+Click on a line to view a specific profile:
+
+{{< img src="tracing/profiling/profiling_flamegraph.gif" alt="A specic profile">}}
+
+The profile header contains information associated with your profile, like the service that generated it, or the environment and code version associated to it.
```

Comment:
```suggestion
The header contains information associated with your profile, like the service that generated it, or the environment and code version associated to it.
```

Reply:
It's a little odd to describe info about this panel that's not visible in the video.